### PR TITLE
Fixed the bug of removing existing labels when add pr labels

### PR DESCRIPTION
**Pull request by sima-zhu**
Tested in this PR: https://github.com/Azure/azure-sdk-for-c/pull/1376

The array gets printed in c Create Pull Request step: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=557006&view=logs&j=96791242-dbf3-587e-3a06-ae5af5c1a705&t=e77f5cb4-719b-54a0-4102-729e63acf003

Fixed issue: https://github.com/Azure/azure-sdk-tools/issues/1046

### CODE REVIEW

File: eng/common/scripts/Submit-PullRequest.ps1 [hunk 1]
```diff
@@ -58,7 +58,7 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
-function AddLabels([int] $prNumber, [string] $prLabelString)
+function AddLabels([int] $prNumber, [string] $prLabelString, [array]$existingLabels)
```
Comment (damirault) [inline eng/common/scripts/Submit-PullRequest.ps1:61]: nit: spacing
```suggestion
function AddLabels([int] $prNumber, [string] $prLabelString, [array] $existingLabels)
```

**Review by damirault — APPROVED** [review]

File: eng/common/scripts/Submit-PullRequest.ps1 [hunk 2]
```diff
@@ -97,7 +103,7 @@ if ($resp.Count -gt 0) {
 
     # setting variable to reference the pull request by number
     Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp[0].number)"
-    AddLabels $resp[0].number $PRLabels
+    AddLabels $resp[0].number $PRLabels $resp.labels
```
Comment (weshaggard) [inline eng/common/scripts/Submit-PullRequest.ps1:106]: should this be `$resp[0].labels`?

Comment (sima-zhu) [reply eng/common/scripts/Submit-PullRequest.ps1:106]: Yes, will fix it.

**Review by weshaggard — APPROVED** [review]
One question but otherwise looks good.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**